lib/inih/ini.c

```diff
@@ -199,11 +199,6 @@ int ini_parse_stream(ini_reader reader, void* stream, ini_handler handler,
                 *end = '\0';
                 name = rstrip(start);
                 value = end + 1;
-#if INI_ALLOW_INLINE_COMMENTS
```

Comment:
https://github.com/prusa3d/Prusa-Firmware-Buddy/blob/master/lib/inih/README.md#compile-time-options

> **Inline comments**: By default, inih allows inline comments with the `;` character. To disable, add `-DINI_ALLOW_INLINE_COMMENTS=0`. You can also specify which character(s) start an inline comment using `INI_INLINE_COMMENT_PREFIXES`.
> 
> **Start-of-line comments**: By default, inih allows both `;` and `#` to start a comment at the beginning of a line. You can override this by changing `INI_START_COMMENT_PREFIXES`.

Did you consider simply tweaking either or both of these options, vs eliding this single block of code?

Reply:
We do not want to remove inline comments. Instead we just want to remove inline comments where inline comments shouldn't happen : On parsing values. That's why only this define is removed, but INI_ALLOW_INLINE_COMMENTS stays defined.

Reply:
Ah okay, sounds like a bug in [inih](https://github.com/benhoyt/inih) then?  Or, is this an effect of the quoted string modification?  I think it's not given the comments from the issue this addresses but unsure.

Perhaps this is fixed in more recent inih?  https://github.com/benhoyt/inih

Reply:
Sadly, it isn't fixed in the latest master, it's an issue by design of the parser. It would require to be rewritten and it would add more complexity. However this is only a temporary fix to solve current user issues until the inih is completely removed and replaced by a different parser library.